assets/scss/core/_variables.scss

```diff
@@ -25,26 +26,33 @@ $dp-font-color-dark: $dp-font-color !default;
 $dp-font-color-white: #fff !default;
 
 
-$dp-font-size-xxl: 60px;
-$dp-headers-h1-font-size: 50px;
-$dp-headers-h2-font-size: 40px;
-$dp-headers-h3-font-size: 30px;
-$dp-headers-h4-font-size: 22px;
-$dp-form-font-size: 20px;
-$dp-headers-h5-font-size: 18px;
-$dp-headers-h6-font-size: 13px;
-$dp-font-size-small: 13px;
-$dp-font-size-xsmall: 11px;
-$dp-font-size-form: 14px;
-$dp-font-button-big: 15px;
-$dp-font-button-medium: 13px;
-$dp-font-button-small: 11px;
-$dp-font-size-15: 15px;
-$dp-font-size-24: 24px;
-$dp-font-small-lineheight: 18px;
+////
+/// Font Size settings
+///
+
+$dp-font-size--lv0: 0;
+$dp-font-size--lv1: 10px;
+$dp-font-size--lv2: 11px;
+$dp-font-size--lv3: 12px;
+$dp-font-size--lv4: 13px;
+$dp-font-size--lv5: 14px;
+$dp-font-size--lv6: 15px;
```

Comment:
Do we use all this fontsizes for real? Seems a bit too much

Reply:
Omg, 19 font sizes? are we using them as other things also?(maybe a rename should be done)

Reply:
I agree as we have standardized spaces for common usages, we should have standardized font sizes for common usages.